packages/oui-dual-list/dual-list.less

```diff
@@ -0,0 +1,237 @@
+@import './_normalize';
+@import './_mixins';
+
+// dual list
+.oui-dual-list {
+  background: @oui-color-dual-list-background;
+  border: 1px solid @oui-color-dual-list-header-background;
+  border-radius: rem-calc(3px);
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+
+  @media (min-width: 600px)  {
+    flex-wrap: nowrap;
+  }
+
+  &___full-height {
+    height: 100%; 
+  }
+
+  &__source {
+    border-right: 1px solid @oui-color-dual-list-header-background;
+  }
+
+  &__source, &__target {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 100%;
+    height: auto;
+  }
+
+  &__header {
+    display: flex;
+    flex-direction: row;
+    background: @oui-color-dual-list-header-background;
+    padding: rem-calc(10px) @oui-dual-list-right-padding;
+    height: rem-calc(40px);
+
+    &_toggle {
+      display: inline-block;
+      margin-left: auto;
+      .oui-icon {
+        font-size: @oui-dual-list-icon-size;
+      }
+      &:active, &:focus {
+        outline: none;
+      }
+      @media (min-width: 600px)  {
+        display: none !important;
+      }
+      @media (min-width: 600px)  {
+        &_up, &_down {
+          display: none;
+        }
+      }
+    }
+  }
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    padding: 0;
+    border-radius: 0 0 rem-calc(3px) rem-calc(3px);
+    height: calc(100% - rem-calc(40px));
+
+    &_filter {
+      position: relative;
+      padding: rem-calc(10px) @oui-dual-list-left-padding;
+      height: rem-calc(60px);
+
+      & > input {
+        background: @oui-color-dual-list-background;
+        border: 1px solid @oui-color-dual-list-header-background;
+        box-shadow: inset rem-calc(2px) rem-calc(2px) 1px #e9ebe7;
+        border-radius: rem-calc(3px);
+        width: 100%;
+        padding-right: 3rem;
+        height: rem-calc(40px);
+        padding-left: rem-calc(10px);
+      }
+
+      & > .oui-icon {
+        position: absolute;
+        right: @oui-dual-list-left-padding + rem-calc(10px);
+        top: 50%;
+        transform: translateY(-50%);
+        font-size: 2rem;
+      }
+    }
+
+    &_add-all, &_remove-all {
+      padding: rem-calc(5px) 0 0 0;
+      padding-left: @oui-dual-list-left-padding;
+      text-decoration: underline;
+      color: @oui-color-dual-list-link;
+      &:hover {
+        cursor: pointer;
+      }
+      &:hover, &:active {
+        color: @oui-color-dual-list-link;
+      }
+    }
+
+    &_remove-all {
+      @media (min-width: 600px)  {
+        padding: rem-calc(5px) @oui-dual-list-left-padding;
+        margin-left: auto;
+      }
+    }
+
+    &_add-all {
+      @media (min-width: 600px)  {
+        padding-left: @oui-dual-list-left-padding + @oui-dual-list-left-padding;
+      }
+    }
+
+    a, a:active, a:hover {
```

Comment:
This seems to be a duplicate from `.oui-link`.
Prefer using base style from our existing classes if possible :)